spec/mercure.md

```diff
@@ -411,12 +283,17 @@ authorized to send updates for the specified topics.
 
 To be allowed to publish an update, the JWS presented by the publisher **MUST** contain a claim
 called `mercure`, and this claim **MUST** contain a `publish` key. `mercure.publish` contains an
-array of topic selectors. See (#topic-selectors).
+array of topic matchers. See (#subscription).
+
+Topic matchers must **MUST** be strings, in which case they will **MUST** be matched exactly by the topic,
+or objects with the a `match` entry containing the topic matcher itself, and an optional `matchType` entry containing the topic matcher
```

Comment:
I did not understood the sentence `will **MUST**`, `with the a match`

Reply:
clarified/fixed